pages/groupe-6.html

```diff
@@ -3,6 +3,55 @@
+
+    <li><h2>Printemps de Bourges du 24 au 29 avril</h2></li>
+
+    <p>Le Printemps de Bourges est un des plus célèbres festivals français avec chaque année la crème de la crème des artistes de variété française, de rock et de musiques électroniques. 6 jours, 6 événements, et au-delà de cela, c’est toute une ville qui bouge au rythme de la musique.</p>
+
+    <li><h2><span><lang=en>Hellfest</lang></span> du 22 au 24 juin</h2></li>
+
+    <p>Doit-on encore présenter le <span><lang=en>Hellfest</lang></span> ? Festival majeur européen dans le monde du Métal, le <span><lang=en>Hellfest</lang></span> réunit chaque année plusieurs dizaines de milliers de festivaliers, et l’année dernière a battu un record puisqu’ils étaient près de 200 000 ! En 2018, le rdv est fixé du 22 au 24 juin.</p>
+
+    <Li><h2><span><lang=en>Solidays</lang></span> du 22 au 24 juin</h2></li>
+
+    <p>Les <span><lang=en>Solidays</lang></span>, deuxième plus grand festival de France revient à l’hippodrome de Longchamp pour son 20ème anniversaire ! Inspiré du mouvement hippie, cet événement rassemble des fans de musiques tous genres afin de les sensibiliser sur une seule thématique : la lutte contre le SIDA. Plus que des concerts, il y a des conférences et de la sensibilisation aux 4 coins des 5 hectares du lieu.</p>
+
+    <li><h2>Beauregard du 6 au 9 juillet</h2></li>
+
+    <p>Le festival Beauregard revient en grande pompe cet été pour son édition 2018. Aux portes de Caen, le festival s’est établi dans le parc du château de Beauregard depuis 2009, pour le plus grand plaisir des normands et des autres !</p>
+
+    <li><h2>Vieilles charrues du 19 au 22 juillet </h2></li>
+
+    <p>Musiques éclectiques, voici le qualificatif majeur de ce monstre des festivals français. Avec une programmation à la pointe, il satisfait toujours tous les styles de musique. Et cette année encore vous ne serez pas en reste puisqu’ont été annoncés de grands noms comme <span><lang=en>Depeche Mode</lang></span>, <span><lang=en>IAM</lang></span>, <span><lang=en>Liam Gallagher</lang></span>, Lomepal, <span><lang=en>Gorillaz</lang></span>, <span><lang=en>Massive Attack</lang></span>, Damso, Roméo Elvis, Lorenzo… rien que ça !</p>
+
+    <li><h2>Lollapalooza du 21 et 22 juillet</h2></li>
+
+    <p>C’est la deuxième année consécutive que ce festival itinérant natif des Etats-Unis s’installe en France et il faut dire qu’en 2017 il a rencontré un carton plein ! <span><lang=en>Lana del Rey</lang></span>, les <span><lang=en>Red Hot Chili Peppers</lang></span> et <span><lang=en>The Weeknd</lang></span> avaient fait le déplacement à l’Hippodrome de Longchamp.</p>
```

Comment:
`<span lang="en">Lana del Rey</span>, les <span lang="en">Red Hot Chili Peppers</span> et <span lang="en">The Weeknd</span>`